blazor/ClientSideExample/ClientSideExample/ViewModels/CounterViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reactive;
+using System.Threading.Tasks;
+using ReactiveUI;
+
+namespace ClientSideExample.ViewModels
+{
+    public class CounterViewModel : ReactiveObject
+    {
+        private int currentCount;
+
+        public CounterViewModel()
+        {
+            Increment = ReactiveCommand.CreateFromTask(IncrementCount);
+        }
+
+        public int CurrentCount => currentCount;
+        
+
+        public ReactiveCommand<Unit, Unit> Increment;
```

Comment:
Can this be a read-only Property instead?

Reply:
It could, certainly, but the idea is to demonstrate using a ReactiveCommand.

Reply:
I think Ana means `public ReactiveCommand<Unit, Unit> Increment { get; }`

Reply:
Done.